cynghanedd.py

```diff
@@ -205,14 +216,57 @@ def choose_word(solutions, word):
 
     words = []
 
-    while solution and solution[0]:
+    words, solution = complete_cyng(words, skeleton, solution)
+
+    return words, skeleton, solution
+
+
+def back_cynghanedd(old_words, skeleton):
+    solution = search(skeleton)
+
+    print(old_words[:-1])
+    words = []
+    i = 0
+    current_iteration = len(old_words)
+    print("Current iteration is ", end="")
```

Comment:
You can remove this and the next line. In future, if you want a shorter way to spell this, you can do:

```
print(f"Current iteration is {current_iteration}")
```